### PR TITLE
PFM-1190 Search approved estimates, mobile view and fixed localizations dropdown

### PR DESCRIPTION
**Pull request by nipunarora-eGov**

### CODE REVIEW

**Review by burhanj-eGov — APPROVED** [review]

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/works/src/components/SearchApprovedSubEstimate/MobileSearchApprovedEstimates.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+            [t("WORKS_CREATED_BY")]: row?.additionalDetails?.owner || t("ES_COMMON_null"),
+            [t("WORKS_OWNER")]: row?.additionalDetails?.owner || t("ES_COMMON_null"),
+            [t("WORKS_STATUS")]: row.estimateStatus || t("ES_COMMON_null"),
+            [t("WORKS_TOTAL_AMT")]: row.amount || t("ES_COMMON_null"),
+            [t("WORKS_ACTIONS")]: (
+                <span className="link">
+                    <Link to={`create-loi?estimateNumber=${row.estimateNumber}&subEstimateNumber=${row.estimateDetailNumber}`}>
+                        <div style={{ "display": "flex", "justifyContent": "flex-start", "alignItems": "center" }}>
+                            <span ><CreateLoiIcon style={{ "margin": "auto" }} />  </span>
+                            <p style={{"marginLeft":"0.5rem"}}>{"Create LOI"}</p>
```
Comment (jagankumar-egov) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/works/src/components/SearchApprovedSubEstimate/MobileSearchApprovedEstimates.js:90]: localise it

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/works/src/components/SearchApprovedSubEstimate/index.js [hunk 2]
```diff
@@ -53,80 +45,81 @@ const SearchApprovedSubEs = ({ tenantId, onSubmit, data, count }) => {
...
             Cell: ({ row }) => {
                 return (
-                    <div>
                         <span className="link">
-                            <Link to={`/digit-ui/employee/`}>
-                                <span ><CreateLoiIcon  style={{ "margin": "auto" }} /> {"Create LOI"} </span>
+                        <Link to={`create-loi?estimateNumber=${row.original.estimateNumber}&subEstimateNumber=${row.original.estimateDetailNumber}`}>
+                                <div style={{"display":"flex","justifyContent":"space-between","alignItems":"center"}}>
+                                    <span ><CreateLoiIcon style={{ "margin": "auto" }} />  </span>
+                                    <p>{"Create LOI"}</p>
```
Comment (jagankumar-egov) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/works/src/components/SearchApprovedSubEstimate/index.js:108]: same here

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/works/src/pages/employee/SearchApprovedSubEstimate.js [hunk 3]
```diff
@@ -124,9 +140,38 @@ const SearchApprovedSubEstimate = () => {
...
+                <Toast
+                error={showToast.error}
+                warning={showToast.warning}
+                label={t(showToast.label)}
+                onClose={() => {
+                    setShowToast(null);
+                }}
+                />
+            )}
+            <SearchApplicationApproved onSubmit={onSubmit} data={getData()} tenantId={"pb.jalandhar"} count={isLoadingEstimateSearch?.length} resultOk={isResultsOk()} isLoading={isLoadingEstimateSearch}/>
```
Comment (jagankumar-egov) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/works/src/pages/employee/SearchApprovedSubEstimate.js:172]: check this tenant id has to be const

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/works/src/components/SearchApprovedSubEstimate/index.js [hunk 4]
```diff
@@ -53,80 +45,81 @@ const SearchApprovedSubEs = ({ tenantId, onSubmit, data, count }) => {
...
         {
             Header: t("WORKS_BUDGET_HEAD"),
-            accessor: (row) => GetCell(row.budgetHead),
+            accessor: (row) => GetCell(t(`ES_COMMON_BUDGETHEAD_${row.budgetHead}`)),
             disableSortBy: true,
         },
         {
             Header: t("WORKS_CREATED_BY"),
-            accessor: (row) => GetCell(row.auditDetails.createdBy),
+            accessor: (row) => GetCell(row?.additionalDetails?.owner || t("ES_COMMON_null")),
```
Comment (jagankumar-egov) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/works/src/components/SearchApprovedSubEstimate/index.js:82]: keep as ES_COMMON_NA

**Review by jagankumar-egov — APPROVED** [review]

**Pull request merged**